core/org.openjdk.jmc.flightrecorder.rules.jdk/src/main/java/org/openjdk/jmc/flightrecorder/rules/jdk/general/JfrPeriodicEventsFixRule.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (c) 2021 Oracle and/or its affiliates. All rights reserved.
+ *
```

Comment:
Should add Datadog copyright line under the Oracle one (see other such files by searching for Datadog).